bin/ch-test

```diff
@@ -1171,6 +1189,28 @@ case $phase in
         done
         echo
         ;;
+    most)
```

Comment:
I wonder if we should run `most` in CI unless full scope? Or maybe we should remove the `most` phase and just skip `61_force_auto.bats` in `all` unless we're in full scope? We'd want to skip the whole file, not individual tests, because skipping 100 tests takes a while.